src/tests/unit/test_save.py

```diff
@@ -233,7 +233,6 @@ def test_save_no_overwrite_uncontained_document(self):
                         "_id": "2",
                         "name": "im_a_uncontained_attribute",
                         "type": "basic_blueprint",
-                        "contained": False,
```

Comment:
Don't think we can remove this. It's used at least when we are deleting stuff. To know if we should delete the "model contained" uncontained child.

Reply:
If an attribute is contained or not is defined in the blueprint, not in the entity, right? The `uncontained_blueprint` defines `uncontained_in_every_way`to be `contained: False`. Adding it here too makes no sense, right?